Find a functionally equivalent code:com.android.internal.policy.impl.PhoneWindowManager.systemReady:COMMENT
Method Modifier: public      internal    
Comment:/**
 * {@inheritDoc}
 */

@@ -1,8 +1,8 @@
 {
-    if (!mHeadless) {
-        mKeyguardDelegate = new KeyguardServiceDelegate(mContext, null);
-        mKeyguardDelegate.onSystemReady();
-    }
+    mKeyguardDelegate = new KeyguardServiceDelegate(mContext, null);
+    mKeyguardDelegate.onSystemReady();
+    readCameraLensCoverState();
+    updateUiMode();
     synchronized (mLock) {
         updateOrientationListenerLp();
         mSystemReady = true;

